Mathlib/Data/List/Basic.lean

```diff
@@ -446,6 +446,19 @@ theorem forall_mem_iff_getElem {l : List α} {p : α → Prop} :
     (∀ x ∈ l, p x) ↔ ∀ (i : ℕ) (_ : i < l.length), p l[i] := by
   simp [mem_iff_getElem, @forall_swap α]
 
+theorem surjective_get_iff {l : List α} : l.get.Surjective ↔ (∀ x, x ∈ l) :=
```

Comment:
I think the naming guide would suggest `get_surjective_iff`, since surjective is called out as special.